src/ethers.ts

```diff
@@ -43,25 +52,24 @@ export class EthersMulticall {
       this.doCalls.bind(this),
       options
     );
+
+    this.defaultBlockTag = defaultBlockTag;
   }
 
-  static async new(
-    provider: ethers.providers.Provider,
-    options: DataLoader.Options<ContractCall, any> = DEFAULT_DATALOADER_OPTIONS
-  ) {
+  static async new(provider: Provider, options?: Omit<Partial<EthersMulticallOptions>, "chainId">) {
     const network = await provider.getNetwork();
 
     return new EthersMulticall(provider, {
+      ...options,
```

Comment:
The naming is a bit confusing since `EthersMulticallOptions` also has a `options` property

Reply:
yes and actually we're getting rid of this in the next PR so we won't have to rename it